meta-rpi-extras/README.md

```diff
@@ -0,0 +1,4 @@
+META-PELUX-BSP-RPI
```

Comment:
I think there is a commit in pyro that changes the name here.

Reply:
Done.